website/docs/docs/cloud/configure-cloud-cli.md

```diff
@@ -103,12 +103,23 @@ The dbt Cloud CLI uses the same set of [dbt commands](/reference/dbt-commands) a
 - Automatically defers build artifacts to your Cloud project's production environment.
 - Supports [project dependencies](/docs/collaborate/govern/project-dependencies), which allows you to depend on another project using the metadata service in dbt Cloud. 
   - Project dependencies instantly connect to and reference (or  `ref`) public models defined in other projects. You don't need to execute or analyze these upstream models yourself. Instead, you treat them as an API that returns a dataset.
-
+ 
 :::tip Use the <code>--help</code> flag
 As a tip, most command-line tools have a `--help` flag to show available commands and arguments. Use the `--help` flag with dbt in two ways:
 - `dbt --help`: Lists the commands available for dbt<br />
 - `dbt run --help`: Lists the flags available for the `run` command
 :::
+ 
+### SQLFluff integration
+The dbt Cloud CLI supports [SQLFluff](https://sqlfluff.com/), a modular and configuration SQL linter, which warns you of complex functions, syntax, formatting, and compilation errors.
+
+To get started, run `dbt sqlfluff -h` to see the list of supported commands and flags, such as `dbt sqlfluff lint` to lint SQL files.
+
+#### Considerations:
+- When you run `dbt sqlfluff`, it picks up changes to your local .sqlfluff config.
```

Comment:
We might want to sprinkle a little more commentary on what this means or why it's a "consideration"

Reply:
this was just launched i believe so i can imagine things will iterate, so i wanted to drop some considerations since the customer facing teams were asking them based on the user questions they received.